### PR TITLE
Adds support to toggle search field

### PR DESCRIPTION
**Pull request by PatrickChrestin**
I was wondering if it should be made possible to toggle the search field.
This would be of the same nature as toggling the copy and curl buttons.
This would be useful on smaller devices.
It is just a minor change.

Feel free to comment and give your opinion

### CODE REVIEW

File: lib/src/ui/widgets/fancy_dio_tab_view.dart [hunk 1]
```diff
@@ -42,21 +42,22 @@ class _FancyDioTabViewState<T extends NetworkBaseModel>
...
-        const Divider(height: 1),
+          const Divider(height: 1),
+        ],
         Expanded(
           child: ListView.separated(
             itemCount: filteredComponents.length,
             separatorBuilder: (context, index) => const Divider(height: 8),
             itemBuilder: (context, index) {
               final filteredComponent = filteredComponents[index];
-
```
Comment (gokhancvs) [inline lib/src/ui/widgets/fancy_dio_tab_view.dart:59]: This is a unnecessary change. We can revert it.

File: lib/src/models/fancy_dio_inspector/fancy_dio_inspector_tile_options.dart [hunk 2]
```diff
@@ -13,8 +13,12 @@ class FancyDioInspectorTileOptions {
   /// then the string will be truncated and `...` will be appended at the end.
   final int? maxLines;
 
+  /// It controls whether to show the search field to filter elements or not.
+  final bool showSearch;
+
   const FancyDioInspectorTileOptions({
-    this.showButtons = true,
+    this.showButtons = false,
```
Comment (gokhancvs) [inline lib/src/models/fancy_dio_inspector/fancy_dio_inspector_tile_options.dart:20]: `showButtons` has to be `true` as default.

**Review by gokhancvs — CHANGES_REQUESTED** [review]
I've two minor change requests for you. 😄

File: lib/src/models/fancy_dio_inspector/fancy_dio_inspector_tile_options.dart (hunk 2 above)
Comment (PatrickChrestin) [reply lib/src/models/fancy_dio_inspector/fancy_dio_inspector_tile_options.dart]: Good catch 👍
